src/popup/popup.js

```diff
@@ -384,25 +384,20 @@ function createTopSiteDetailEls(originData, originStats) {
 				textEl(`Crunching time spent numbers...`)
 			]);
 		}
+	} else {
+		setContentOfEl(timeSpentEl, [textEl(`Crunching time spent numbers...`)]);
 	}
 
-	// Set visit count text
-	let visitCountText = ``;
 	const visitCount = originData.originVisitData.numberOfVisits;
-	if (visitCount === 0) {
-		// Don't set the visit count text
-	} else if (visitCount === 1) {
-		visitCountText = `${visitCount} time`;
-	} else {
-		visitCountText = `${visitCount} times`;
-	}
+
+	// Set visit count text
 	let visitCountEl = paragraphEl();
-	if (visitCountText !== ``) {
+	if (visitCount > 0) {
 		const percentVisits = getPercentVisitsToOriginOutOfTotal(originData, originStats);
 		if (percentVisits > 0) {
 			setContentOfEl(visitCountEl, [
 				textEl(`You've visited `),
-				strongEl(visitCountText),
+				strongEl(`${visitCount} time${visitCount > 1 ? `s` : `` }`),
```

Comment:
nit for readability:

I think pulling out the check to add the `s` to `time` would make this a bit more readable.